app/helpers/application_helper/toolbar/storage_resource_center.rb

```diff
@@ -0,0 +1,2 @@
+class ApplicationHelper::Toolbar::StorageResourceCenter < ApplicationHelper::Toolbar::Basic
```

Comment:
@llivne @skateman why do we need these empty toolbars?

Reply:
My guess is that without them the toolbar chooser crashes...but I'm not 100% sure.

Reply:
It was a while ago, but I think that without it you get crashes. we do intend to add toolbar buttons for storage resource in the near future, I can check @h-kataria suggestion to remove the code from app/helpers/application_helper/toolbar_chooser.rb but we do want to add buttons so I guess its up with you if you want to do it now or only when I add buttons.

Reply:
@llivne it's better to add the toolbars along with buttons, if it works with the suggestion above otherwise i am good with adding toolbars in this PR. But we do need Product features for sure to go with this PR, i did not see a link to PR that is adding those in core repo maybe i missed that.

Reply:
@h-kataria ok so i tried it and it worked. I removed the toolbars files (required me to also modify toolbar_chooser.rb like you suggested). About the Product feature, storage resources is a new feature for all block storages so it goes with the existing block storage feature and no need for new one. Currently only the new autosde block storage manager will support it, and for other block storage managers like cinder you will just get an empty list for it, but hopefully in the future they will also add it to the provider since it's a feature that all storages have (sometime with different names like pools. This is why we used a generic name, storage resources)

Reply:
@h-kataria I had another thought about the storage resource features, and I guess that if we will add for example an add storage resource button, we will need a feature for supporting add storage resource so that only supported block storages can do it, but this will be handled once we implement the buttons for it, and I dont think its part of this PR.